pkgs/development/python-modules/iterfzf/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  fzf,
+  packaging,
+  pytestCheckHook,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "iterfzf";
+  version = "1.4.0.51.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "dahlia";
+    repo = "iterfzf";
+    rev = "refs/tags/${version}";
+    hash = "sha256-Xjk+r2PdIg+oULA5gfI129p1fNOO8RINNxP+pveiocM=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail 'dynamic = ["version"]' 'version = "${version}"' \
+      --replace-fail 'backend-path = ["."]' '# backend-path = ["."]' \
+      --replace-fail 'build-backend = "build_dist"' '# build-backend = "build_dist"'
+
+    substituteInPlace iterfzf/test_iterfzf.py \
+      --replace-fail 'executable="fzf"' 'executable="${fzf}/bin/fzf"'
+  '';
+
+  build-system = [
+    flit-core
+    setuptools
+    packaging
+  ];
+
+  dependencies = [ fzf ];
```

Comment:
This shouldn't be propagated. Instead, the source code should be patch and the path to fzf hardcoded.